docs/_docs/Starters Academy (LDSSA)/06-Remote-Experience.md

```diff
@@ -0,0 +1,139 @@
+presents itself, be proactive and collaborative in coming up with a solution. Invite someone that 
+interests you to a private video chat. If there's too many people in a specific room for it to 
+be useful, create another one and invite specific people to it. You'll spend less time on the 
+learning material but you'll spend more time problem solving with other people. Problem solving 
+with people is how real relationships are built. Having relationships with people in the course 
+are what will carry you through when you're doubting yourself. 
+
+There are going to be technical difficulties. We are a volunteer organization and this is the 
+first time we've done this. When there are problems, make it an opportunity to meet someone 
+somehow and treat it as an ice breaker. In doing this, you'll make a friend or two and that 
+friend or two will make the next 9 months so much better.
+
+
+
+## The tools
+
+### The google calendar 
+
+The source of truth for the schedule is [the batch 5 google calendar](https://calendar.google.com/calendar/u/2?cid=Y18yODE2azVnYXVscmw3MXVybnJwbGhjbW05c0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t). 
+You can use the link itself directly or you can add it to your favorite calendar app. 
+
+### Google meet (or google hangouts, same thing)
+
+We will use google meet for the entire bootcamp. If this works we'll continue using it for the 
+rest of the course. We've got a zoom subscription ready to go in the unlikely case this does not 
+work. Make sure that you are prepared to use both google meet AND zoom so that we are able to 
+quickly adapt should the need arise!
+
+### The auditorium room
+
+The auditorium room is the room from which all presentations will be given. This is the big room 
+that everyone can enter when someone needs to speak to everyone else. All SLU presentations and 
+all important announcements will be given from this room and only this room.
+
+[This is the auditorium room](https://meet.google.com/qne-ypod-msq). As with all rooms, when you 
+join, someone will need to let you in. If you see the option appear on your screen to let someone 
+join, do it!
+
+### Side rooms
+
+Side rooms are specific to SLUs, BLUs, Hackathons, and even coffee breaks. On the google calendar, 
+there are links for each of these. If you want to enter a side room, you can leave the auditorium 
+and go into the side room. Easy-peasy. Just don't remember to go back to the auditorium before the 
+next lecture starts!
+
+### Slido
+
+We have all been in that work meeting or class where that one person forgets to mute themselves, 
+making a huge mess of the meeting. Managing so many people in a meeting and making sure everyone 
+gets a turn is a big challenge. To avoid confusion and give everyone a chance to clarify the 
+presentation content or just ask more about a topic, the Q&A this year will make use of slido 
+(https://www.sli.do/).
```

Comment:
are we still using this?

Reply:
I would advise so, it is useful and we can prepare things before hand